src/commands/shared.ts

```diff
@@ -30,19 +32,33 @@ import yargs from "yargs";
  *  to be configured
  */
 const GRANT_TIMEOUT_MILLIS = 60e3;
+// The prefix of installed SSH accounts in P0 is the provider name
+const SUPPORTED_PROVIDERS = ["aws", "gcloud"];
```

Comment:
small nit: maybe these keys could use a type like SshRequest["type"] to prevent misspelling or using an invalid provider?